lib/patch_ruby/api/estimates_api.rb

```diff
@@ -93,6 +94,70 @@ def create_bitcoin_estimate_with_http_info(create_bitcoin_estimate_request, opts
       return data, status_code, headers
     end
 
+    # Create an ethereum estimate given a timestamp and gas used
+    # Creates an ethereum estimate for the amount of CO2 to be compensated. An order in the `draft` state may be created based on the parameters, linked to the estimate. 
+    # @param create_ethereum_estimate_request [CreateEthereumEstimateRequest] 
+    # @param [Hash] opts the optional parameters
+    # @return [EstimateResponse]
+    def create_ethereum_estimate(create_ethereum_estimate_request, opts = {})
+      data, _status_code, _headers = create_ethereum_estimate_with_http_info(create_ethereum_estimate_request, opts)
+      data
+    end
+
+    # Create an ethereum estimate given a timestamp and gas used
+    # Creates an ethereum estimate for the amount of CO2 to be compensated. An order in the &#x60;draft&#x60; state may be created based on the parameters, linked to the estimate. 
```

Comment:
assuming this is an html encoding bug with openapi but there are `&#x60;` chars in this comment

Reply:
Good catch. Let me check the swagger file to see if the issue is there or is due to code generation.